### PR TITLE
driver/bluetooth: Implement the null driver for bluetooth

### PR DESCRIPTION
**Pull request by amitpurwar**
Signed-off-by: amit purwar <amit.purwar@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [88a27887186538d4b47b676e7d020154a7b28793] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [88a27887186538d4b47b676e7d020154a7b28793] - Code Rule Check (C++) OK.

File: os/drivers/wireless/Kconfig [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-endif # DRIVERS_WIRELESS
-
+if DRIVERS_WIRELESS
+&& SELECT_SCSC_WLAN source drivers / wireless / scsc / Kconfig endif
+#DRIVERS_WIRELESS
+if DRIVERS_WIRELESS
+	&& SELECT_RTK_WLAN source drivers / wireless / realtek / Kconfig endif
+#DRIVERS_WIRELESS
+	if DRIVERS_WIRELESS
+		&& BLUETOOTH source drivers / wireless / bluetooth / Kconfig endif
```
Comment (Wootak) [inline os/drivers/wireless/Kconfig]: Please do not change other parts except bluetooth.
Below configure is fine.
--------------------------------------------
if DRIVERS_WIRELESS && BLUETOOTH

source drivers/wireless/bluetooth/Kconfig

endif # DRIVERS_WIRELESS

**Comment by Wootak** [comment]
Dear Amit,
Please fix below warning points.
I think we need to export "bt_driver_register" function to include/tinyara/bluetooth/ header.
Thank you.

CC:  wireless/bluetooth/bt_null.c
wireless/bluetooth/bt_null.c: In function 'btnull_process_write_local_name_req':
wireless/bluetooth/bt_null.c:215:3: warning: implicit declaration of function 'wlinfo' [-Wimplicit-function-declaration]
   wlinfo("CMD: %04x\n", opcode);
   ^~~~~~
wireless/bluetooth/bt_null.c: In function 'btnull_send':
wireless/bluetooth/bt_null.c:241:4: warning: implicit declaration of function 'wlerr' [-Wimplicit-function-declaration]
    wlerr("ERROR: Failed to allocate buffer\n");
    ^~~~~
wireless/bluetooth/bt_null.c: In function 'btnull_register':
wireless/bluetooth/bt_null.c:309:9: warning: implicit declaration of function 'bt_driver_register' [-Wimplicit-function-declaration]
  return bt_driver_register(&g_bt_null);
         ^~~~~~~~~~~~~~~~~~
CC:  dev_null.c

**Comment by seinfra** [comment]
Target : [dda87c1b1870c4a4929a8f6f4cb93628badab763] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [dda87c1b1870c4a4929a8f6f4cb93628badab763] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [85e4ada6e2d16c8e748dc5e501329791a68691d7] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [85e4ada6e2d16c8e748dc5e501329791a68691d7] - Code Rule Check OK.

**Comment by Wootak** [comment]
Looks good to me.

File: os/drivers/wireless/bluetooth/Make.defs [hunk 2]
```diff
@@ -0,0 +1,45 @@
+############################################################################
+# drivers/bluetooth/Make.defs
+#
```
Comment (pillip8282) [inline os/drivers/wireless/bluetooth/Make.defs]: samsung license is required

File: os/drivers/wireless/bluetooth/bt_null.c [hunk 3]
```diff
@@ -0,0 +1,310 @@
+/****************************************************************************
+ * drivers/wireless/bluetooth/bt_null.c
+ * UART based Bluetooth driver
```
Comment (pillip8282) [inline os/drivers/wireless/bluetooth/bt_null.c]: samsung license is required. and if source is not come from nuttx. I think you don't need to write nuttx license

**Comment by seinfra** [comment]
Target : [cca9bfefed5a2ebb615cffdd43db6bedcd338be0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [cca9bfefed5a2ebb615cffdd43db6bedcd338be0] - Code Rule Check OK.

**Pull request merged**